### PR TITLE
improvement(configuration): Search ami_id_db_oracle by version in region

### PR DESCRIPTION
**Pull request by aleksbykov**
Gemini test use ami_id_db_oracle hard coded in test config for specified region
This cause test run failed if gemini test run on another region that ami set
With new config parameter 'oracle_scylla_version'  ami_id_db_oracle will be choose
by region

Trello: https://trello.com/c/MK1Zea3Q
## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [ ] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [x] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [x] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/sct_config.py [hunk 1]
```diff
@@ -168,6 +168,12 @@ class SCTConfiguration(dict):
                      Automatically lookup AMIs and repo links for formal versions.
                      WARNING: can't be used together with 'scylla_repo' or 'ami_id_db_scylla'"""),
 
+        dict(name="oracle_scylla_version", env="SCT_ORACLE_SCYLLA_VERSION",
```
Comment (fruch) [inline sdcm/sct_config.py]: please recreate the docs:
`SCT_CLUSTER_BACKEND=aws hydra conf-docs -o markdown > docs/configuration_options.md`

Comment (aleksbykov) [reply sdcm/sct_config.py]: Done

File: docs/configuration_options.md [hunk 2]
```diff
@@ -1,185 +1,201 @@
...
-| **<a name="post_behavior_monitor_nodes">post_behavior_monitor_nodes</a>**  | Failure/post test behavior, i.e. what to do with the monitor cloud instances at the end of the test.<br><br>'destroy' - Destroy instances and credentials (default)<br>'keep' - Keep instances running and leave credentials alone<br>'keep-on-failure' - Keep instances if testrun failed | keep-on-failure | SCT_POST_BEHAVIOR_MONITOR_NODES
-| **<a name="workaround_kernel_bug_for_iotune">workaround_kernel_bug_for_iotune</a>**  | Workaround a known kernel bug which causes iotune to fail in scylla_io_setup, only effect GCE backend | N/A | SCT_WORKAROUND_KERNEL_BUG_FOR_IOTUNE
-| **<a name="loader_swap_size">loader_swap_size</a>**  | Swap file size in bytes calculated by x * 1MB | 1024 | SCT_LOADER_SWAP_SIZE
+Docker version 19.03.5, build 633a0ea838
```
Comment (fruch) [inline docs/configuration_options.md]: can you clean this section ? the whole hydra stuff shouldn't be here...

Comment (bentsi) [reply docs/configuration_options.md]: @aleksbykov please remove hydra logging info

**Pull request merged**